views/index.njk

```diff
@@ -65,6 +65,7 @@
                 <li><strong>Express Server:</strong> Fast and lightweight web framework</li>
                 <li><strong>TypeScript:</strong> Type-safe development environment</li>
                 <li><strong>Static File Serving:</strong> CSS, JS, and asset management</li>
+                <li><strong><a href="/job-roles" style="color: #3498db; text-decoration: none;">Job Roles:</a></strong> Browse available job opportunities</li>
```

Comment:
👀 This styling should be in a css file and then use a class like `.job-roles-link` if you'll have it in multiple places, or an id `#job-role-link` if it'll exist just once here